test/over_react/component/dom_components_test.dart

```diff
@@ -25,15 +25,20 @@ import 'package:over_react/over_react.dart';
 import 'package:react/react_client.dart';
 import 'package:test/test.dart';
 
-
 main() {
   group('Dom component:', () {
-    ClassMirror domClassMirror = reflectClass(Dom);
-
-    List<MethodMirror> methods = domClassMirror.staticMembers.values.toList();
+    List<MethodMirror> methods = [];
+    ClassMirror domClassMirror;
 
     setUpAll(() {
-      expect(methods, isNotEmpty, reason: 'should have properly reflected the static members.');
+      domClassMirror = reflectClass(Dom);
+
+      // staticMembers is not implemented for the DDC and will throw is this test is loaded even if it's not run.
+      try {
```

Comment:
I don't know if it's evaluated, I think the DDC might just be checking for instances of that being used, because it throws if it's in a test too